utils/build/docker/nodejs/express/app.js

```diff
@@ -448,10 +448,12 @@ app.get('/flush', (req, res) => {
   // does have a callback :)
   const promises = []
 
-  const { profiler } = require('dd-trace/packages/dd-trace/src/profiling/')
-  if (profiler?._collect) {
-    promises.push(profiler._collect('on_shutdown'))
-  }
+  try {
+    const { profiler } = require('dd-trace/packages/dd-trace/src/profiling/')
+    if (profiler?._collect) {
+      promises.push(profiler._collect('on_shutdown'))
+    }
+  } catch {}
```

Comment:
would it be good to add a trace in the catch?

Reply:
```suggestion
  } catch (err) {
    console.error('Unable to flush profiler', err)
  }
```